pyflakes/checker.py

```diff
@@ -322,10 +322,11 @@ class Binding(object):
                 the node that this binding was last used.
     """
 
-    def __init__(self, name, source):
+    def __init__(self, name, source, isUnassigned=False):
```

Comment:
this is a bit of a double-negative and a [boolean trap](https://youtu.be/CnRkXO_a5mI) -- it would probably be best to change this to `assigned=True` and make it keyword-only

Reply:
sure! should i wait for https://github.com/PyCQA/pyflakes/pull/660 to be merged before introducing a keyword-only arg?

Reply:
yeah I just merged it